src/store/getters/state-getters.ts

```diff
@@ -94,7 +94,7 @@ export const isShowFilePayBtn = (state: any): boolean => {
  */
 export const isEnableFilePayBtn = (state: any): boolean => {
   const step1Valid = state.stateModel.defineCompanyStep.valid
-  const step2Valid = true // FOR TESTING ONLY
+  const step2Valid = state.stateModel.addPeopleAndRoleStep.valid // FOR TESTING ONLY
```

Comment:
You can remove the "TESTING" comment now.

Reply:
removed